source/administration/console/managing-deployment.rst

```diff
@@ -0,0 +1,109 @@
+
+.. _minio-console-managing-deployment:
+
+========================
+Managing Your Deployment
+========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The MinIO Console provides a graphical user interface to monitor or manage your deployment.
```

Comment:
:thinking: y'know in avoiding the passive 'is' i sometimes wonder if we go too far.

"The MinIO Console is a graphical user interface that supports monitoring and managing your MinIO deployment."

feels fine and a little less wordy?

Reply:
Dropping "graphical user interface" on all the pages and reworking the intro sentences on each.